src/lib.rs

```diff
@@ -65,6 +78,18 @@ impl From<&NixFile> for PathBuf {
     }
 }
 
+impl ToString for NixFile {
+    fn to_string(&self) -> String {
+        PathBuf::from(self).as_path().to_string_lossy().to_string()
```

Comment:
I don’t think we can have this instance, where is it used? Filenames are `[u8]` on unix, we shouldn’t convert like this.

Reply:
src/daemon/internal_proto.rs#320 (also L331 and 343)

Reply:
How should we convert otherwise?